tools/super-tox.py

```diff
@@ -476,50 +497,66 @@ def main(
         handlers=[rich.logging.RichHandler()],
     )
 
-    if mode in ("local", "lxd-per-tox"):
+    assert mode in ("local", "lxd", "lxd-per-tox")
```

Comment:
Is this for the type checker? click should be enforcing this already.

Reply:
Yep, it's either this or a cast.
I'd rather not `#type :ignore`, if I can help it.